src/atomic.rs

```diff
@@ -784,9 +784,11 @@ impl<T> AsMut<T> for Owned<T> {
 #[derive(Debug)]
 pub struct Ptr<'scope, T: 'scope> {
     data: usize,
-    _marker: PhantomData<&'scope T>,
+    _marker: PhantomData<(&'scope (), *const T)>,
 }
 
+unsafe impl<'scope, T: Send> Send for Ptr<'scope, T> {}
```

Comment:
Do we need this?

Reply:
Yes, because `defer` requires `Send`, so for `Ptr` to be usable with `defer` we do need it. That said, raw pointer aren't `Send`, so maybe this is a bit wierd, but it's meant to be used in concurrent contexts.

Reply:
Yes. Since `Ptr` now contains a `*const T`, it will not be `Send` by default and you won't be able to pass it into `scope.defer(...)`. For that reason, we must manually implement `Send`.